hermes/bin/src/runtime_extensions/hermes/http_gateway/routing.rs

```diff
@@ -174,3 +180,64 @@ fn compose_http_event(
         HTTPEventMsg::HTTPEventReceiver => Ok(error_response("HTTP event msg error".to_owned())?),
     }
 }
+
+/// Serves static data with 1:1 mapping
+fn serve_static_data(path: &str) -> anyhow::Result<Response<Body>> {
+    let vfs = VFS
+        .get()
+        .ok_or(anyhow::anyhow!("Unable to obtain virtual filesystem"))?;
+
+    let file = vfs.read(path)?;
+
+    Ok(Response::new(file.into()))
+}
+
+/// Check if valid path to static files.
+fn is_valid_path(path: &str) -> anyhow::Result<bool> {
```

Comment:
What do you think to even change the return type of this function to this.
So in the case of the invalid path error would be returned
```suggestion
fn is_valid_path(path: &str) -> anyhow::Result<()> {
```